package.json

```diff
@@ -39,6 +39,7 @@
     "sync-request": "3.0.1",
     "typescript": "1.8.7",
     "validator": "3.33.0",
-    "node-uuid": "1.4.6"
+    "node-uuid": "1.4.6",
+    "nock": "9.0.2"
```

Comment:
This introduces a different problem. Requiring dev dependencies to be global is a limitation if multiple tasks require different versions of the same package.